app/values/scanned_resource_pdf/canvas.rb

```diff
@@ -1,5 +1,7 @@
 class ScannedResourcePDF
   class Canvas
+    LETTER_WIDTH = 612
```

Comment:
You know, I don't really know what the scale factor is doing to those images if it's in points. I was assuming pixels too, but you're right.